File: webservice/classes/note.php
<?php


Class Note{
    
    private $_id;
    private $_author;
    private $_text;
    private $_date;
    private $_theZone;
    
    function GetId() {
        return $this->_id;
    }

    function GetAuthor() {
        return $this->_author;
    }

    function GetText() {
        return $this->_text;
    }

    function GetDate() {
        return $this->_date;
    }

    function GetTheZone() {
        return $this->_theZone;
    }

    function SetId($id) {
        $this->_id = $id;
    }

    function SetAuthor($author) {
        $this->_author = $author;
    }

    function SetText($text) {
        $this->_text = $text;
    }

    function SetDate($date) {
        $this->_date = $date;
    }

    function SetTheZone($theZone) {
        $this->_theZone = $theZone;
    }


    
    function __construct($_id = null, $_author = null, $_text = null, $_date = null, $_theZone = null) {
        $this->_id = $_id;
        $this->_author = $_author;
        $this->_text = $_text;
        $this->_date = $_date;
        $this->_theZone = $_theZone;
    }

    
    
}

File: ios-client/Exametrics/Exametrics/Note.swift
//
//  Note.swift
//  Exametrics
//
//  Created by Vilaine Emilien on 19/05/2016.
//  Copyright © 2016 IMERIR. All rights reserved.
//

import Foundation
import RealmSwift
import Realm

class Note : Object{
    
    // Champs
    private dynamic var _id     : String = ""
    private dynamic var _author : String = ""
    private dynamic var _text   : String = ""
    private dynamic var _date   : String = ""
    private dynamic var _idArea : String = ""
    
    // Getter & Setter
    func getId() -> String{
        return _id
    }
    
    func setId(id: String) {
        _id = id
    }
    
    func getAuthor() -> String{
        return _author
    }
    
    func setAuthor(author: String) {
        _author = author
    }
    
    func getText() -> String{
        return _text
    }
    
    func setText(text: String) {
        _text = text
    }
    
    func getDate() -> String{
        return _date
    }
    
    func setDate(date: String) {
        _date = date
    }
    
    func getIdArea() -> String{
        return _idArea
    }
    
    func setIdArea(idArea: String) {
        _idArea = idArea
    }
    
    
    // Constructeurs
    func setNote(id: String, author: String, text: String, date: String, idArea : String){
        
        _id = id
        _author = author
        _text = text
        _date = date
        _idArea = idArea
    }
    
}